### PR TITLE
Validation of collector input args

### PR DESCRIPTION
**Pull request by slemrmartin**
Input arguments of source in multisource mode are not tested in `bin script`. They should be tested before creating Thread with collector

- [x] **depends on** https://github.com/ManageIQ/topological_inventory-ingress_api-client-ruby/pull/51

### CODE REVIEW

**Comment by slemrmartin** [comment]
@Ladas pls :eyes:

**Review by Ladas — APPROVED** [review]
:+1:

File: lib/topological_inventory/ansible_tower/collectors_pool.rb [hunk 1]
```diff
@@ -18,6 +18,16 @@ def path_to_secrets
       File.expand_path("../../../secret", File.dirname(__FILE__))
     end
 
+    def source_valid?(source, secret)
```
Comment (Ladas) [inline lib/topological_inventory/ansible_tower/collectors_pool.rb:21]: this should be extended to actually validate the credentials, correct?

Comment (slemrmartin) [reply lib/topological_inventory/ansible_tower/collectors_pool.rb:21]: it validates if credentials are present, they are in `secret`. Or do you mean validations that credentials are valid? I think this is similar to Source availability checker, which implements @abellotti - possibly if it'll be in db, then that source won't be event deployed (after merge of that)

**Pull request merged**